src/overlays/misc/ovl_kaleido_scope/z_kaleido_equipment.c

```diff
@@ -170,8 +170,8 @@ void KaleidoScope_DrawEquipment(PlayState* play) {
             while (cursorMoveResult == 0) {
                 if (pauseCtx->stickAdjX < -30) {
                     if (pauseCtx->cursorX[PAUSE_EQUIP] != 0) {
-                        pauseCtx->cursorX[PAUSE_EQUIP] -= 1;
-                        pauseCtx->cursorPoint[PAUSE_EQUIP] -= 1;
+                        pauseCtx->cursorX[PAUSE_EQUIP]--;
+                        pauseCtx->cursorPoint[PAUSE_EQUIP]--;
```

Comment:
not a fan of ++/-- on `cursorPoint`, it happens to be 1 here because the layout is row-major, but when moving vertically it gets offset by 4
For consistency I'd keep `+-= 1` for it

also in `z_kaleido_item.c`